pox-locking/src/events.rs

```diff
@@ -337,6 +337,15 @@ fn create_event_info_data_code(function_name: &str, args: &[Value]) -> String {
                 pox_addr = &args[3],
             )
         }
+        "revoke-delegate-stx" => {
+            format!(
+                r#"
+                {{
+                    data: {{ }}
```

Comment:
Can this include the same set of properties as the corresponding `delegate-stx` event? At a minimum the `delegate-to` property which would make it easier to expose these events to the associated pool operators

Reply:
That data is not available during the call. It can be called while delegating or while non-delegating

Reply:
Seems like arguably a bug or at least unexpected behavior in pox-4. Consumers of this event will have to do extra work to determine if the revocation call/event was actually meaningful. 

The `(map-delete delegation-state ..)` here should not return success if no change occurred (i.e. check if the map has something to delete first) https://github.com/stacks-network/stacks-core/blob/2c1c777b9ed0124a65e9cd2c019bd85f0735a5e0/stackslib/src/chainstate/stacks/boot/pox-4.clar#L636-L641

Reply:
I haven't seen a requirement for meaningful or non-meaningful revoke txs. My experience says it is ok like this. We only want to have the tx-sender.

Reply:
I created an issue for discussion https://github.com/stacks-network/stacks-core/issues/4146

Reply:
moved to https://github.com/stacks-network/stacks-core/pull/4157